File: app/src/main/java/darkpoint/rubel/CustomChip.java
package darkpoint.rubel;

import android.content.Context;

import com.google.android.material.chip.Chip;

public class CustomChip extends Chip {
    private int SELECTED_ID;

    public int getSELECTED_ID() {
        return SELECTED_ID;
    }

    public void setSELECTED_ID(int SELECTED_ID) {
        this.SELECTED_ID = SELECTED_ID;
    }

    public CustomChip(Context context) {
        super(context);
    }
}

File: app/src/main/java/darkpoint/rubel/customViews/CustomChip.kt
package darkpoint.rubel.customViews

import android.content.Context

import com.google.android.material.chip.Chip
import darkpoint.rubel.activities.CurrenciesIndex
import darkpoint.rubel.model.Rate

class CustomChip(context: Context) : Chip(context) {
    var selectedID: Int = 0
    lateinit var curIndex: CurrenciesIndex
    lateinit var rate: Rate
}
